fang/mysql_migrations/migrations/2023-08-17-102017_create_fang_tasks/up.sql

```diff
@@ -3,17 +3,23 @@
 
 -- docker exec -ti mysql mysql -u root -pfang -P 3360 fang -e "$(catn fang/mysql_migrations/migrations/2023-08-17-102017_create_fang_tasks/up.sql)"
 
+ /*
+        why `metadata` and `error_message` are not a TEXT ?
+        MySQL TEXT type, I think it is stored as a BLOB.
+        So that breaks FromRow trait, implemented in lib.rs line 183
+    */
+
 CREATE TABLE fang_tasks (
     id VARCHAR(36) DEFAULT (uuid()) PRIMARY KEY,
-    metadata JSON NOT NULL,
-    error_message TEXT,
+    metadata VARCHAR(2048) NOT NULL,
+    error_message VARCHAR(2048),
     state ENUM('new', 'in_progress', 'failed', 'finished', 'retried') NOT NULL DEFAULT 'new',
     task_type VARCHAR(255) NOT NULL DEFAULT 'common', -- TEXT type can not have default value, stupid MySQL policy
-    uniq_hash CHAR(64),
+    uniq_hash VARCHAR(64),
     retries INTEGER NOT NULL DEFAULT 0,
-    scheduled_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+    scheduled_at VARCHAR(32) NOT NULL DEFAULT(CONCAT(CURRENT_TIMESTAMP, '.000000000+00')),
```

Comment:
why timestamps are stored as varchars?

Reply:
The main reason compatibility with `SQLite` as this DBMS treats almost everything as string. Also the format of the date is not decoded as expected.

Reply:
maybe each backend can define its own way to convert the record from their databse type to a common struct?

Reply:
will be addressed in issue #148